packages/orbit-design-tokens/src/index.ts

```diff
@@ -5,6 +5,7 @@ export const defaultTheme = createTheme();
 
 export { default as fromPlainObject } from "./fromPlainObject";
 export { default as transparentColor } from "./js/transparentColor";
+export type { Tokens as paletteColors } from "./js/paletteColors";
```

Comment:
remove that from index.ts as export, that's why you are getting that error on flow ci :)

Reply:
I can't remove. I need it, that is precisely why I created it…

Reply:
I did not mean to remove completely, just to remove from index file as export. Already did it for you.